KZL-557 - Add collection controller csharp (#191)
kuzzleio/documentation-V2: src/guide/1/getting-started/first-steps/index.md

curl "http://localhost:7512/?pretty"
```

> If everything is working you should see a JSON document containings a list of API endpoints.

Reviewer: ```suggestion
If everything is working you should see a JSON document containing a list of API endpoints.
```

// instantiate a Kuzzle client, this will automatically connect to the Kuzzle server
const kuzzle = new Kuzzle('localhost', {defaultIndex: 'playground'})

> // create an object containings the message we want to store

Reviewer: ```suggestion
// creates an object containing the message we want to store
```

const message = {message: "Hello, World!"}

> // create a document in the 'mycollection' collection

Reviewer: ```suggestion
// creates a document in the 'mycollection' collection
```
